tests/FakeIngest.php

```diff
@@ -130,7 +130,7 @@ public function decodedWrites(): Collection
 
     public function forgetWrites(): void
     {
-        $this->streams->forget($this->streams->keys());
+        $this->streams->pop($this->streams->count());
```

Comment:
This fixes an issue where subsequent writes would not write to the 0th position.